src/components/Buttons/__tests__/RadioGroup.spec.js

```diff
@@ -48,37 +48,39 @@ describe('RadioGroup', () => {
     const onChange = sinon.spy()
     const wrapper = mount(
       <RadioGroup name="radioTest" onChange={onChange}>
-        <Radio id="1" value="val1">Value 1</Radio>
-        <Radio id="2" value="val2">Value 2</Radio>
+        <Radio id="A" value="val1">Value 1</Radio>
+        <Radio id="B" value="val2">Value 2</Radio>
       </RadioGroup>
     )
 
-    wrapper.find('#1').simulate('change')
-    expect(onChange.calledOnce).toBe(true)
+    expect(onChange.callCount).toBe(0)
+
+    wrapper.find('input#A').simulate('change', { target: { checked: true } })
```

Comment:
I tried to simulate `'click'`, but apparently that does not work ([[1](https://github.com/airbnb/enzyme/issues/1201)] [[2](https://github.com/airbnb/enzyme/issues/216#issuecomment-191114107)])